follow_followers_bot/following_look_up.py

```diff
@@ -0,0 +1,54 @@
+from dotenv import load_dotenv
+load_dotenv()
+import requests
+import os
+import json
+
+bearer_token = os.environ.get("Bearer_tokken")
+
+def create_url(following_user_id):
+    user_id = following_user_id
+    return f"https://api.twitter.com/2/users/{user_id}/following"
+
+
+def get_params():
+    return {"user.fields": "created_at"}
+
+
+def bearer_oauth(r):
+    """
+    Method required by bearer token authentication.
+    """
+
+    r.headers["Authorization"] = f"Bearer {bearer_token}"
+    r.headers["User-Agent"] = "v2FollowingLookupPython"
+    return r
+
+
+def connect_to_endpoint(url, params):
+    response = requests.request("GET", url, auth=bearer_oauth, params=params)
+    if response.status_code != 200:
+        raise Exception(
+            "Request returned an error: {} {}".format(
+                response.status_code, response.text
+            )
+        )
+    return response.json()
+
+
```

Comment:
Why are you having same functions in multiple files. The whole idea of creating function was to write them once at some place and reuse them at multiple places.